rekcurd/core/rekcurd_worker.py

```diff
@@ -44,7 +44,7 @@ def predict(self, predictor: object, idata: PredictInput, option: dict = None) -
         raise NotImplemented()
 
     @abstractmethod
-    def evaluate(self, predictor: object, filepath: str) -> Tuple[EvaluateResult, List[EvaluateResultDetail]]:
+    def evaluate(self, predictor: object, filepath: str) -> Generator[EvaluateResultDetail, None, EvaluateResult]:
```

Comment:
I am not familiar with `Generator`, so please tell me.
Why is the second factor is `None`?

Reply:
Generator type hinting is formed with `Generator[YieldType, SendType, ReturnType]` . 
(generator type has `send` method (ref: https://stackoverflow.com/questions/45627872/python3-send-function-in-generators ))
, but I do not use `send` in this case, so I leave it `None`

Reply:
Thank you for your clarification!